main.go

```diff
@@ -376,7 +378,20 @@ func handle(root string) func(w http.ResponseWriter, req *http.Request) {
 		}
 		text = renderIndex(names)
 		wg.Done()
+
+		watcher, err = internal.NewWatcher(func(string) {
```

Comment:
I thought it is for development convenience. Not sure if it's needed under production environment.

Reply:
In go-zero's community, many developers asked for the file watch and auto reload functionality.

But to keep it simple, I don't think it's worth to add it. Usually I told them to use `air` instead.

For Go+, I have the same opinion. I think maybe it's better to support auto reload in a third party tool.

Reply:
> I think maybe it's better to support auto reload in a third party tool.

I agree.

I mean, for tutorial site, should we disable this functionality under production environment? As it may introduce uncertainty, such as

* Will `main.go` panic if we update gop code on server which does not suit previous version of `main.go` (still runing)? As there are some string operations with gop code in `main.go`
* Is fsnotify efficient enough on different OS?

Reply:
I agree to disable it on production.

I don't think performance is a problem for tutorial project.

Reply:
> I don't think performance is a problem for tutorial project.